source/features/add-toggle-files-button.js

```diff
@@ -0,0 +1,18 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate';
+import * as icons from '../libs/icons';
+
+export default function () {
+	select('.commit-tease .float-right').appendChild(<button
+		class="btn-octicon p-1 pr-2 js-toggle-files"
+		aria-label="Toggle files section"
+		aria-expanded="true">
+		{icons.arrowUp()}
+		{icons.arrowDown()}
+	</button>);
+	const repoContent = select('.repository-content');
+	delegate('.js-toggle-files', 'click', ({currentTarget}) => {
+		currentTarget.setAttribute('aria-expanded', !repoContent.classList.toggle('files-hidden'));
```

Comment:
Instead of toggling a class, I'd like to just use the `hidden` property like in my example in #598.

Reply:
I tried a lot of different ways to solve it and this turned out to be the one with the smallest complexity. If it was only the `.files` list that had to be modified, I'd go with `hidden`, but since CSS doesn't have a "previous sibling" selector, this would require more code.

Reply:
Gotcha, you're right